data/json/monsters.json

```diff
@@ -210,7 +210,7 @@
     "vision_day":40,
     "hp":16,
     "death_function":"NORMAL",
-    "description":"A spotted wild cat living across much of North America.  It is not a serious threat to humans, but it can be aggressive if not left alone.",
+    "description":"A spotted wild cat, these live across much of North America.  It is not a serious threat to humans, but it can be aggressive if not left alone.",
```

Comment:
```"cat ... these live"``` confuses me a bit (singular & plural forms in one sentence). I'm not a native speaker, so maybe it's my mistake and sentence is actually correct, so just let me know if everything is okay here.